rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -59,6 +59,7 @@ def _precompiled_apple_resource_bundle_impl(ctx):
             config_vars = ctx.var,
             device_families = ["iphone", "ipad"],
             explicit_minimum_os = None,
+            explicit_minimum_deployment_os = None,
```

Comment:
I assume this is required due to the `rules_apple` bump right?

Reply:
It is [a new feature that landed 2 weeks ago](https://github.com/bazelbuild/rules_apple/pull/1099) that we want to leverage and expose to rules_ios callers, but does effectively exactly what Jerry was looking to do to decouple minimum OS version from from the version used to compile the code.